### ISSUE TITLE
File not being created on Mac

### ISSUE THREAD

**Issue by nealculiner** [banditoth/maui-appleprivacy-vscode#2]
Hello, I press CMD + SHIFT + P and then type in Privacy, choose the option to create a new privacy file and select the project. I see the folder structure created but it shows file is missing, it's not getting written to disk. I'm using VSCode on M1 Mac. Any suggestions on how to resolve this?

**Comment by banditoth** [comment]
Hello @nealculiner ,

Are you sure that you are providing input for this question?
<img width="461" alt="image" src="https://github.com/user-attachments/assets/a64cafba-3366-4e06-bdf4-29fdff68347f">

Doesn't it give you error message like this?
<img width="477" alt="image" src="https://github.com/user-attachments/assets/5932d807-52f0-49b9-bc32-69be13118d66">

**Comment by nealculiner** [comment]
I tried answering both yes and no to the first dialog, I never see the second. I'm using a prior Xamarin.iOS app converted to net8-ios so I'm not really using maui but it should still work. I see the entry in the .csproj file but no file is written to the path specified. So again, no to the second dialog, never see it.

**Comment by banditoth** [comment]
@nealculiner 
Is it creating the Platforms/iOS folder structure correctly?

**Comment by nealculiner** [comment]
Yes, just the file itself is not written to disk it seems. I see the folder structure but not file.

**Comment by banditoth** [comment]
Is it adding the propertygroup tag at the very end of your .csproj file?

**Comment by nealculiner** [comment]
Yes, all that looks fine, just no file created.

**Comment by banditoth** [comment]
As a workaround, consider creating a new MAUI app with the Maui Extension for VSCode, and run the command on the newly created project. I will look into this.
Is your source code public by any chance?

**Comment by nealculiner** [comment]
I will try and report back

Private code

**Comment by nealculiner** [comment]
@banditoth it works in a test .NET Maui app so I'll copy the file over to my project. Thanks.

**Comment by nealculiner** [comment]
I don't know if there is a path/file structure you're looking for. As it's a straight iOS app, not .net maui structure with platforms maybe that's throwing your add-on off? I pasted in the file, updated the .csproj and then used your tool to edit the file and it wouldn't find it. Not sure if there is a way to allow the user to select their file regardless of where it's located, I put mine in the root, or do a search for the specific extension within the project, or read it from the .csproj file, and allow the user to edit it wherever they store it?
